cmd/multi-factor-auth/config/api.toml

```diff
@@ -15,11 +15,11 @@ RestApiInterface = ":8080" # The interface `address and port` to which the REST
 
 [APIPackages.guardian]
     Routes = [
-        { Name = "/register", Open = true, Auth = true },
-        { Name = "/sign-message", Open = true, Auth = false },
-        { Name = "/sign-transaction", Open = true, Auth = false },
-        { Name = "/sign-multiple-transactions", Open = true, Auth = false },
-        { Name = "/verify-code", Open = true, Auth = true },
+        { Name = "/register", Open = true, Auth = true , MaxContentLength = 500 },
+        { Name = "/sign-message", Open = true, Auth = false, MaxContentLength = 500 },
+        { Name = "/sign-transaction", Open = true, Auth = false, MaxContentLength = 500 },
```

Comment:
this should be increased, perhaps 2000?

reason: for this transaction :point_down:  , which is a simple one with not much data field, the total size is 576:
```
{
	"code": "774040",
	"transaction": {
		"nonce": 0,
		"value": "0",
		"receiver": "erd10ksryjr065ad5475jcg82pnjfg9j9qtszjsrp24anl6ym7cmeddshwnru8",
		"sender": "erd1at9keal0jfhamc67ulq4csmchh33eek87yf5hhzcvlw8e5qlx8zq5hjwjl",
		"gasPrice": 1000000000,
		"gasLimit": 56000,
		"data": "dGVzdA==",
		"signature": "77534cc11b6c894cad2b308ab8917cd233394afa459255bcea2c8c28e71b310b7d591e794df80270f050ee212d7f42fc4045f9cb92a9f9cf26fdab28d7214708",
		"chainID": "T",
		"version": 2,
		"options": 2,
		"guardian": "erd1t9ws78h9f0letqqsxnzrrpl7tw50w79u5rl4kdw0k5ugkhp0lr3q8um8v5"
	}
}
```